### ISSUE TITLE
Wrong image selection position

### ISSUE THREAD

**Issue by Kirilych** [xdan/jodit#8]
![wrong_image_selection](https://user-images.githubusercontent.com/5024436/33230498-2ec13550-d1ed-11e7-819e-fec7e498fe61.png)
simple test variant:
```
<div style="width:800px; margin:auto; border:1px solid red;">
	wrong image selection
	<div style="position:relative;">
		<textarea id="text_area0"> <img src="tigr.jpg" style="width:100px;"/> </textarea>
		<script>
			var object = document.getElementById("text_area0");
			new Jodit(object);
		</script>
	</div>
</div>
```

bootstrap variant:  (with bootstrap.min.css)
```
<div class="container">
<div class="row">
	<div class="col-md-12">
		wrong image selection
		<textarea id="text_area"> <img src="tigr.jpg" style="width:100px;"/> </textarea>
		<script>
			var object = document.getElementById("text_area");
			new Jodit(object);
		</script>
	</div>
</div>
```

**Comment by xdan** [comment]
Hi thank you fixed in 3.0.36

**Comment by xdan** [comment]
Can i close it?
